src/test/java/reposense/model/StandaloneConfigTest.java

```diff
@@ -0,0 +1,54 @@
+package reposense.model;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import org.junit.Test;
+
+import reposense.parser.StandaloneConfigJsonParser;
+import reposense.template.GitTestTemplate;
+
+public class StandaloneConfigTest extends GitTestTemplate {
+
+    private static final Path VALID_CONFIG = new File(StandaloneConfigTest.class.getClassLoader()
+            .getResource("StandaloneConfigTest/valid_config.json").getFile()).toPath();
+    private static final Path INVALID_GITID_CONFIG = new File(StandaloneConfigTest.class.getClassLoader()
+            .getResource("StandaloneConfigTest/invalidGitId_config.json").getFile()).toPath();
+    private static final Path INVALID_DISPLAYNAME_CONFIG = new File(StandaloneConfigTest.class.getClassLoader()
+            .getResource("StandaloneConfigTest/invalidDisplayName_config.json").getFile()).toPath();
+    private static final Path INVALID_ALIASES_CONFIG = new File(StandaloneConfigTest.class.getClassLoader()
+            .getResource("StandaloneConfigTest/invalidAuthorAliases_config.json").getFile()).toPath();
+    private static final Path INVALID_IGNOREGLOB_CONFIG = new File(StandaloneConfigTest.class.getClassLoader()
+            .getResource("StandaloneConfigTest/invalidIgnoreGlob_config.json").getFile()).toPath();
+
+    @Test
+    public void standaloneConfig_validJson_success() throws IOException {
+        StandaloneConfig standaloneConfig = new StandaloneConfigJsonParser().parse(VALID_CONFIG);
+        config.update(standaloneConfig);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void standaloneConfig_invalidGitIdJson_throwIllegalArgumentException() throws IOException {
```

Comment:
Since we are already adding tests for `StandaloneConfig` already, can we add additional tests, like non-json file, contain invalid field names?

Reply:
that should belong to `StandaloneConfigJsonParserTest` and we can add that in other PR in future